README.md

```diff
@@ -47,6 +47,14 @@ docker:
     - "/path/to/another-image-name.tar"
 ```
 
+### Run a Command
+device-init can execute a list of commands on boot. e.g.:
+```yaml
+runcmd:
+  - "apt-get install package-name"
```

Comment:
The user is providing the device-init.yaml. So, he is responsible that the yaml is correct and can be parsed. 

The important part is that it is easy for the end user to write command lines without too much yaml-escaping.